examples/AutoScrollExample.js

```diff
@@ -29,13 +29,14 @@ class AutoScrollExample extends React.Component {
         {rowIndex}, {columnKey}{' '}
       </div>
     );
+    const headerRenderer = ({ columnKey }) => <div> {columnKey} </div>;
 
     for (let i = 0; i < 100; i++) {
       this.columns[i] = (
         <Column
           key={i}
           columnKey={i}
-          header={<div> {i} </div>}
+          header={headerRenderer}
```

Comment:
Unrelated to this PR, but I grabbed it along the way.
Passing a div React element causes errors on the console cause FDT was passing non-div props (like isVisible, columnKey, etc). This slightly decreases performance in dev mode.